src/components/ArticleCard.jsx

```diff
@@ -0,0 +1,16 @@
+const ArticleCard = ({article}) => {
+      return (    <section className="art_cards_main">
+                            <li>
+                            <h4>Topic:{article.topic}</h4>
+                            <p>Title :{article.title}</p>
```

Comment:
I would make the title of your article as the heading as that is what should stand out to the user